ktor-server/ktor-server-plugins/ktor-server-default-headers/nix/src/io/ktor/server/plugins/defaultheaders/DefaultHeaders.nix.kt

```diff
@@ -0,0 +1,29 @@
+/*
+* Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+*/
+
+package io.ktor.server.plugins.defaultheaders
+
+import kotlinx.cinterop.*
+import platform.posix.*
+
+@OptIn(ExperimentalForeignApi::class)
+internal actual fun readVersion(): String {
+    memScoped {
+        val versionFilePath = "/nix/resources/ktor-version.txt"
```

Comment:
it looks like it will not work if the user doesn't have "ktor-version.txt" file in the application resources